assets/src/blocks/CarouselHeader/CarouselHeaderBlock.js

```diff
@@ -36,6 +36,10 @@ const attributes = {
       return {isValid, messages};
     },
   },
+  currentImageIndex: {
```

Comment:
Don't we need to add this new attribute in the PHP definition of the block as well?